src/components/ProgressBar/ProgressBar.js

```diff
@@ -45,9 +54,10 @@ class ProgressBar extends Component<Props> {
 }
 
 const Wrapper = styled.div`
-  position: relative;
+  position: ${props => (props.position === 'top' ? 'absolute' : 'relative')};
```

Comment:
I think I'd rather that the ProgressBar just fill its parent container. It feels weird to me that the progress bar controls its own positioning. I'd do something like this instead:

```js
// in LoadingScreen.js
<ProgressBarWrapper>
  <ProgressBar />
</ProgressBarWrapper>

const ProgressBarWrapper = styled.div`
  position: absolute;
  top: 0;
left: 0;
right: 0;
`
```

Reply:
OK, that's better. I've modified it & removed the `position` prop.